### PR TITLE
update for Fastify v5

### PR DESCRIPTION
**Pull request by gurgunday**
Ref: https://github.com/fastify/fastify/issues/5116

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ on:
 
 jobs:
   test:
-    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v3
+    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v4.0.0
```
Comment (Fdawgs) [inline .github/workflows/ci.yml:20]:
```suggestion
    uses: fastify/workflows/.github/workflows/plugins-ci.yml@v4
```

If you use v4 then it'll automatically pick up the latest, i.e. if we release 4.0.1 then it'll use that.

Comment (gurgunday) [reply .github/workflows/ci.yml:20]: Thanks, will do that

But then I'm not understanding something

Why didn't this work with `rate-limit`? Using v4 there with the redis workflow resulted in errors

https://github.com/fastify/fastify-rate-limit/pull/350
https://github.com/fastify/fastify-rate-limit/actions/runs/7195862716

Comment (gurgunday) [reply .github/workflows/ci.yml]: There you go, it didn't work...

I think it might be because `v4` doesn't exist as a release/tag

Comment (Fdawgs) [reply .github/workflows/ci.yml:20]: Ah yeah, forgot the v4 release went awry: https://github.com/fastify/workflows/pull/107, grumble grumble grumble.

File: signer.js [hunk 2]
```diff
@@ -45,12 +43,13 @@ function _sign (value, secret, algorithm) {
...
     throw new TypeError('Cookie value must be provided as a string.')
   }
+
   return value + '.' + crypto
     .createHmac(algorithm, secret)
     .update(value)
     .digest('base64')
     // remove base64 padding (=) as it has special meaning in cookies
-    .replace(base64PaddingRE, '')
+    .replaceAll('=', '')
```
[the remark this answers is not in the document]
Comment (gurgunday) [reply signer.js:52]: Wow, excuse the silent change, I thought it was an easy win, seems like RegExp is indeed faster

Will revert

**Pull request merged**